src/mongo_client.ts

```diff
@@ -645,9 +649,9 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> {
     }
 
     const session = this.startSession(options);
```

Comment:
again - can we use maybeCallback here?

Reply:
Sure, we needed an extra overload to support this, and I used null to make the usages distinct